lib/standards/html-elms.js

```diff
@@ -647,10 +647,10 @@ const htmlElms = {
 		contentTypes: ['phrasing', 'flow'],
 		allowedRoles: true,
 		// 5.6 output Element
-		namingMethods: 'subtreeText'
+		namingMethods: ['subtreeText']
 	},
 	p: {
-		contentTypes: ['phrasing', 'flow'],
```

Comment:
Why this change?

Reply:
like `kdb` misspelling, it's a correct to the file as 'p' is not phrasing content.